src/csv/builder.cr

```diff
@@ -72,21 +72,22 @@ class CSV::Builder
   # :nodoc:
   def quote_cell(value)
     append_cell do
-      @io << '"'
+      @io << @quote_char
       value.each_byte do |byte|
         case byte
-        when '"'
-          @io << %("")
+        when @quote_char
+          @io << @quote_char
+          @io << @quote_char
```

Comment:
IMO `@io << @quote_char << @quote_char` in one line is clearer

Reply:
fair 'nuff, changed.

Reply:
no????

the two are exactly the same, `<<` just returns `self`.

Reply:
God I wish I had a desktop to benchmark on instead of a laptop, the results are always terribly noisy.